wolfcrypt/src/misc.c

```diff
@@ -546,6 +546,37 @@ WC_STATIC WC_INLINE byte ctSetLTE(int a, int b)
 }
 #endif
 
+WC_STATIC WC_INLINE size_t wc_strlcpy(char *dst, const char *src, size_t dstSize)
```

Comment:
I would prefer to not have these as misc.c inline functions. Just put them in wc_port.c with the other stdlib wc_ type functions. Model like `wc_strtok` and `USE_WOLF_STRTOK`

Reply:
yes, that's better.  I was uneasy about misc.c and it caused me some tribulations in fact.